Fixes Issue #120

etagwerker opened the pull request:
Hey,

This PR fixes #120 with something that was missed on #118

Please check it out.

Thanks!

etagwerker committed:
Fixes https://github.com/DatabaseCleaner/database_cleaner-active_record/issues/120

etagwerker commented:
@mlarraz this should fix the error you reported, could you please test this in your project?

pat reviewed (approved):
Looks good to me! Appreciate the quick patch :)

mlarraz commented:
Yes that appears to have fixed it for me

davisinfo commented:
Can we have a new version with this patch please?

etagwerker committed:
Add a reference to https://github.com/DatabaseCleaner/database_cleaner-active_record/pull/121

etagwerker committed:
Add a reference to https://github.com/DatabaseCleaner/database_cleaner-active_record/pull/121

etagwerker committed:
Patch test expectation

etagwerker committed:
Merge branch 'main' into fixes-120-pre-count-truncate-with-options

etagwerker committed:
[Feature][DX] Add Docker Configuration for Development (#109)

* Fixes https://github.com/DatabaseCleaner/database_cleaner-active_record/issues/120

* Add docker configuration for development

This makes it easier to run the test suite in your local dev environment

* Add a development section for contributors

This should make it easier for people to set up the library and run the test suite within Docker

* Added ruby service to make it easier to get started with your contribution

* Update password to match Docker configuration

* Removed outdated docker-compose file

* Use older mysql to avoid trilogy error, fix docker compose and instructions, mount code for easier development

* Update readme to point to CONTRIBUTE.md file

* Run ./bin/setup so that we have the config file in place for tests

* Relax dependency

* Add a line about the Docker + DX change

---------

Co-authored-by: Ariel Juodziukynas <arieljuod@gmail.com>

nerdrew committed:
release_connection after cleaning (#122)

For Rails >= 7.2, this library gets a connection via
`#lease_connection`. But it never releases the connection. Release the
connection back to the pool after `#clean`-ing.

Rails >= 7.2 does connection health checks / reconnect on connection
checkout. If the connection is never checked back into the pool, it will
never get "repaired", which makes testing database connection issues
tricky.

etagwerker committed:
Change expectation to allow for a cascade option

etagwerker committed:
Merge branch 'main' into fixes-120-pre-count-truncate-with-options

etagwerker commented:
v2.2.2 has been released with this fix: https://rubygems.org/gems/database_cleaner-active_record/versions/2.2.2

taralbass commented:
Greatly appreciated @etagwerker !!!
